src/sql/base/parts/editableDropdown/browser/dropdown.ts

```diff
@@ -301,7 +308,7 @@ export class Dropdown extends Disposable {
 	}
 
 	private _inputValidator(value: string): IMessage | null {
-		if (this._dataSource.options && !this._dataSource.options.find(i => i.value === value)) {
+		if (!this._input.hasFocus() && !this._tree.isDOMFocused() && this._dataSource.options && !this._dataSource.options.find(i => i.value === value)) {
```

Comment:
So the real change here is just no validate if the tree or input has focus? Seems fine as a stepping stone but doesn't seem like a good final solution

Reply:
yes, just like i mentioned in my email, this is temporary solution to fix the regression